AdapterInterface.php

```diff
@@ -53,28 +57,32 @@ public function createAutoRoute(UriContext $uriContext, $document, $tag);
      * Return the canonical name for the given class, this is
      * required as somethimes an ORM may return a proxy class.
      *
+     * @param string $className
+     *
      * @return string
      */
     public function getRealClassName($className);
 
     /**
-     * Return true if the content associated with the auto route
-     * and the given content object are the same.
+     * Compares the content associated with the auto route and the
+     * given content object.
+     *
+     * @param AutoRouteInterface $autoRoute
+     * @param object             $contentObject
      *
-     * @param RouteObjectInterface
-     * @param object
+     * @return bool
```

Comment:
what does compare mean? this could be a bit more specific... equality? or whether the autoRoute and that contentObject belong together?